phpstan new release 1.7.2 fails CI

```
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing phpstan/phpstan (1.7.2): Extracting archive
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=2G --configuration=./phpstan.neon --level=0 apps core settings lib/private lib/public ocs ocs-provider
 1466/1466

 -- ------------------------------------------------------------------------------------------------------------- 
     Error                                                                                                        
 -- ------------------------------------------------------------------------------------------------------------- 
     Child process error (exit code 255): PHP Fatal error:  Cannot declare class Laminas\Validator\EmailAddress,  
     because the name is already in use in                                                                        
     /home/phil/git/owncloud/core/lib/composer/laminas/laminas-validator/src/EmailAddress.php on line 31          
                                                                                                                  
     Child process error (exit code 255): PHP Fatal error:  Cannot declare class Laminas\InputFilter\Factory,     
     because the name is already in use in                                                                        
     /home/phil/git/owncloud/core/lib/composer/laminas/laminas-inputfilter/src/Factory.php on line 12             
                                                                                                                  
 -- ------------------------------------------------------------------------------------------------------------- 


                                                                                                                     
 [ERROR] Found 2 errors                                                                                  
```

If I pin to phpstan 1.7.1 then it passes.

--- a/vendor-bin/phpstan/composer.json
+++ b/vendor-bin/phpstan/composer.json
@@ -1,5 +1,5 @@
 {
     "require": {
-        "phpstan/phpstan": "1.7.1"
+        "phpstan/phpstan": "1.7.7"
     }
 }

